mod/thewire/start.php

```diff
@@ -38,9 +38,6 @@ function thewire_init() {
 	// Extend system CSS with our own styles, which are defined in the thewire/css view
 	elgg_extend_view('elgg.css', 'thewire/css');
 
-	// Add a user's latest wire post to profile
-	elgg_extend_view('profile/status', 'thewire/profile_status');
```

Comment:
This is a content change, not just differences in markup. master is the place to make this change.